presto-spi/src/main/java/io/prestosql/spi/connector/MaterializedViewNotFoundException.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.spi.connector;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+public class MaterializedViewNotFoundException
```

Comment:
This class doesn't belong in this commit. It's not related to "Connector APIs for materialized views that modeled them as TableHandles are being changed to pass in materialized view name rather than handles."

Reply:
Moving to iceberg connector commit